src/main/java/it/pagopa/pn/bff/mappers/notification/NotificationStatusMapper.java

```diff
@@ -0,0 +1,12 @@
+package it.pagopa.pn.bff.mappers.notification;
+
+import it.pagopa.pn.bff.generated.openapi.server.v1.dto.NotificationStatus;
+import org.mapstruct.Mapper;
+import org.mapstruct.factory.Mappers;
+
+@Mapper
+public interface NotificationStatusMapper {
```

Comment:
can we merge this mapper with NotificationReceivedMapper and NotificationSentMapper?